versions.sh

```diff
@@ -87,13 +87,18 @@ for version in "${versions[@]}"; do
 	for suite in \
 		bullseye \
 		buster \
+		alpine3.18 \
 		alpine3.17 \
 		alpine3.16 \
 	; do
 		# https://github.com/docker-library/php/pull/1348
 		if [ "$rcVersion" = '8.0' ] && [[ "$suite" = alpine* ]] && [ "$suite" != 'alpine3.16' ]; then
 			continue
 		fi
+		# https://github.com/docker-library/php/pull/1405
```

Comment:
With no force pushes, this is impressive - you rolled the dice and won :joy: :heart: